executeCommand.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+import subprocess
+
```

Comment:
> Run *user-selected* command on many servers (user provided as param) ...

Hardcoded list of hostnames to work on.
Hardcode simple command to run, no arguments passed to the command to execute on remote host